src/ResourceManager/Reservations/Commands.Reservations/Cmdlets/GetCatalog.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Commands.Reservations.Common;
+using Microsoft.Azure.Management.Reservations.Models;
+using Microsoft.Azure.Commands.Reservations.Models;
+using System.Management.Automation;
+using Microsoft.Azure.Management.Reservations;
+
+namespace Microsoft.Azure.Commands.Reservations.Cmdlets
+{
+    [Cmdlet(VerbsCommon.Get, "AzureRmReservationCatalog"), OutputType(typeof(List<PSCatalog>))]
+    public class GetCatalog : AzureReservationsCmdletBase
+    {
+        [Parameter(Mandatory = false)]
+        [ValidateNotNull]
+        public string SubscriptionId { get; set; }
```

Comment:
It should be possible to pipe in the output of get-azurermsubscription to this cmdlet.  Taking an InputObject of type IAzureSubscription would allow this. (this could be done in the future)

Reply:
There will be lot more fixes that we aim in next sprint, but since our launch is next week we would like to minimize further changes. We appreciate your suggestion and will put that into fixes for next sprint

Reply:
It is required that you support these scenarios, and this was communicated during the cmdlet review.

Reply:
Agreed.  This can be added in the future.